### PR TITLE
Resolved issue with missing header file after project rename

### PR DESCRIPTION
**Pull request by mvanbeusekom**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Bug fix (solves issue #17)

### :arrow_heading_down: What is the current behavior?

When you add the current package to a Flutter project and run `flutter build ios --release` you'll get an error message saying the file `geolocator/geolocator-Swift.h` could not be found.

### :new: What is the new behavior (if this is a feature change)?

Error is fixed

### :boom: Does this PR introduce a breaking change?

No

### :bug: Recommendations for testing

Add the package to a flutter project and run the command `flutter build ios --release`

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Rebased onto current develop

### CODE REVIEW

File: ios/geolocator.podspec [hunk 1]
```diff
@@ -4,13 +4,13 @@
 Pod::Spec.new do |s|
   s.name             = 'geolocator'
   s.version          = '0.0.1'
-  s.summary          = 'Geolocation plugin for Flutter'
+  s.summary          = 'A new flutter plugin project.'
```
Comment (martijn00) [inline ios/geolocator.podspec]: Dit klopt niet helemaal denk ik

File: example/android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Tue Jun 05 16:33:35 CEST 2018
+#Fri Jun 23 08:50:38 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (martijn00) [inline example/android/gradle/wrapper/gradle-wrapper.properties]: Is dit niet een downgrade?

File: android/build.gradle [hunk 3]
```diff
@@ -23,21 +23,12 @@ apply plugin: 'com.android.library'
...
-    buildToolsVersion '28.0.0'
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
 }
-
-dependencies {
-    api 'com.android.support:support-v4:27.1.0'
-    api 'com.google.android.gms:play-services-location:15.+'
```
Comment (martijn00) [inline android/build.gradle]: Je hebt deze dependency toch nodig?

File: ios/geolocator.podspec (hunk 1 above)
Comment (mvanbeusekom) [reply ios/geolocator.podspec]: Ga ik oplossen

File: example/android/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (mvanbeusekom) [reply example/android/gradle/wrapper/gradle-wrapper.properties]: Dit verschil wordt veroorzaakt door de manier waarop je het project genereert (Android Studio vs. Flutter CLI). Ik heb nu de laatste gebruikt en daar komt dit uitrollen.

File: android/build.gradle (hunk 3 above)
Comment (mvanbeusekom) [reply android/build.gradle]: Dat is gek, ik heb ze er hier lokaal wel inzitten. Ga ik even uitzoeken

File: ios/geolocator.podspec [hunk 4]
```diff
@@ -3,14 +3,14 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'geolocator'
-  s.version          = '0.0.1'
-  s.summary          = 'Geolocation plugin for Flutter'
+  s.version          = '0.0.2'
+  s.summary          = 'Geolocation plugin for Flutter.'
   s.description      = <<-DESC
-Geolocation plugin for Flutter
+A new flutter plugin project.
```
Comment (martijn00) [inline ios/geolocator.podspec]: @mvanbeusekom hier missen nog wat gegevens

Comment (mvanbeusekom) [reply ios/geolocator.podspec]: @martijn00 heb het aangevuld.

**Comment by woshitom** [comment]
Hi,
I'm still getting

> Xcode's output:
> ↳
>     === BUILD TARGET location_permissions OF PROJECT Pods WITH CONFIGURATION Debug ===
>     /Users/Logs/Desktop/woshitom/flutter/.pub-cache/hosted/pub.dartlang.org/geolocator-4.0.2/ios/Classes/GeolocatorPlugin.m:2:9: fatal error: 'geolocator/geolocator-Swift.h' file not found
>     #import <geolocator/geolocator-Swift.h>
>             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     1 error generated.
> 
> Could not build the application for the simulator.
> Error launching application on iPhone Xʀ.

even after created a new flutter project

**Pull request merged**